.github/workflows/main.yml

```diff
@@ -75,3 +75,21 @@ jobs:
       with:
         name: artifacts
         path: ./out/artifacts
+
+  build-freebsd:
+    runs-on: macos-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Build
+      id: test
+      uses: vmactions/freebsd-vm@v0.1.4
+      with:
+        usesh: true
+        mem: 8192
+        prepare: |
+          pkg install -y git gtar python3 glib meson pkgconf gobject-introspection \
+            json-glib gnutls gtk-doc vala sqlite3 curl gcab libarchive libelf \
+            gettext-tools gnutls usbids polkit
+        sync: rsync
```

Comment:
if not saving artifacts, I don't know that you need to rsync stuff out do you?

Reply:
Not a clue; I cargo culted that without understanding what it did. @Asiderr ?!

Reply:
Yes, it's needed only when you're saving artifacts.